corehq/ex-submodules/casexml/apps/phone/caselogic.py

```diff
@@ -597,23 +627,27 @@ def filter_cases_modified_elsewhere_since_sync(cases, last_sync):
         # { case_id: [{'token': 'token value', 'date': 'date value'}, ...]}
         all_case_updates_by_sync_token = defaultdict(list)
         for row in modification_dates:
-            # incoming format is a list of objects that look like this:
+            # format from couch is a list of objects that look like this:
             # {
             #   'value': '2012-08-22T08:55:14Z', (most recent date updated)
-            #   'key': ['[case id]', '[sync token id]']
+            #   'key': ['case-id', 'sync-token-id']
             # }
             if row['value']:
-                all_case_updates_by_sync_token[row['key'][0]].append(
-                    {'token': row['key'][1], 'date': datetime.strptime(row['value'], '%Y-%m-%dT%H:%M:%SZ')}
-                )
+                modification_date = datetime.strptime(row['value'], '%Y-%m-%dT%H:%M:%SZ')
```

Comment:
@dannyroberts I just moved this, though wondering whether I should change to use `string_to_utc_datetime` (or `iso_string_to_datetime`?)